component/src/main/java/com/celements/photo/service/ImageScriptService.java

```diff
@@ -301,5 +302,15 @@ public boolean addSlidesFromTemplate(DocumentReference galleryDocRef,
   public String fixMaxImageSizes(String pageContent, int maxWidth, int maxHeight) {
     return maxImageSizeService.fixMaxImageSizes(pageContent, maxWidth, maxHeight);
   }
+  
+  public List<Attachment> getRandomImages(String fullName,
+      int num) throws ClassNotFoundException{
```

Comment:
Why throw an exception? As far as I could see WebUtils.getInstance().getRandomImages doesn't throw an exception. In any case, wouldn't it be better to catch the exception and do something with it instead of throwing it to the frontend?